.gitignore

```diff
@@ -1,22 +1,17 @@
-bin
-build
-/develop-eggs/
-/eggs
-dist
-lib
-parts
-/src/zope.testrunner
-.eggs
-.tox
+*.egg-info
+*.profraw
+*.pyc
+*.pyo
 .coverage
 .installed.cfg
 .mr.developer.cfg
+.tox/
+bin/
 coverage.xml
-nosetests.xml
-htmlcov
-
-*.dll
-*.pyc
-*.pyo
-*.so
-*.egg-info
+develop-eggs/
+dist/
+htmlcov/
+lib/
+lib64/
```

Comment:
I only found this out today: Sometimes `venv` will create `lib64` as a link instead of a folder. That link won't be covered by excluding `lib64/` as folder.